### PR TITLE
Charts: update 'Scale Breaks' demo description

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: JSDemos/Demos/Charts/ScaleBreaks/description.md [hunk 1]
```diff
@@ -1 +1,7 @@
-Scale breaks allow breaking off parts of the axis to improve the readability of a chart with high amplitude values. This example demonstrates auto-calculated scale breaks. You can change the number of breaks, their style or disabled them.
\ No newline at end of file
+A scale break is a wavy stripe on a chart plotting area. It is used to display the break in continuity between too low and too high values on an axis. Scale breaks improve the readability of chart sections with large gaps in their ranges.
```
Comment (Abadzhev) [inline JSDemos/Demos/Charts/ScaleBreaks/description.md]: I wanted to add some explanation about what we mean by improved readability. Some values may look the same (indistinguishable) on a chart without scale breaks. But once you add scale breaks, you can easily compare those values. 

```suggestion
Scale breaks (wavy stripes you see on the chart) cut out ranges from the value axis.  This technique may enhance chart readability if values differ greatly. As you can see in this demo, you can easily compare smaller values if scale breaks are enabled. If you disable scale breaks, the smaller values are indistinguishable. 
```

File: JSDemos/Demos/Charts/ScaleBreaks/description.md [hunk 2]
```diff
@@ -1 +1,7 @@
...
+
+The Chart component can generate scale breaks of the [valueAxis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/). The Chart detects large gaps between side-by-side points, cuts them out, and puts scale breaks instead. Scale breaks are available for *'continuous'* or *'logarithmic'* [type](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/#type) axes only.
```
Comment (Abadzhev) [inline JSDemos/Demos/Charts/ScaleBreaks/description.md]: Just a couple of small corrections. 

```suggestion
The Chart component can generate scale breaks on the [valueAxis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/). The Chart detects large gaps between side-by-side points, cuts them out, and puts scale breaks instead. Scale breaks are available only for *'continuous'* or *'logarithmic'* [axis types](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/#type).
```

File: JSDemos/Demos/Charts/ScaleBreaks/description.md [hunk 3]
```diff
@@ -1 +1,7 @@
...
+
+To enable auto-calculated scale breaks, set the [autoBreaksEnabled](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/#autoBreaksEnabled) property to `true`. Use the [maxAutoBreakCount](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/#maxAutoBreakCount) property to limit the amount of automatically created scale breaks. To configure scale break appearance, use the [breakStyle](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/breakStyle/) object. You can test all these options in the demo.
```
Comment (Abadzhev) [inline JSDemos/Demos/Charts/ScaleBreaks/description.md]: "Amount" usually refers to uncountable nouns. For countables, use "number".

```suggestion
To enable auto-calculated scale breaks, set the [autoBreaksEnabled](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/#autoBreaksEnabled) property to `true`. Use the [maxAutoBreakCount](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/#maxAutoBreakCount) property to limit the number of automatically created scale breaks. To configure scale break appearance, use the [breakStyle](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/breakStyle/) object. You can test all these options in the demo.
```

File: JSDemos/Demos/Charts/ScaleBreaks/description.md [hunk 4]
```diff
@@ -1 +1,7 @@
...
+
+You can also create custom breaks, including breaks of the [argumentAxis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/). Use the [breaks](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/breaks/) array to declare a scale break collection. Each object in this array must have the [startValue](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/breaks/#startValue) and [endValue](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/breaks/#endValue) fields that limit a single scale break. A scale break range should be at least two tick intervals. Otherwise, the break might not be visible.
```
Comment (Abadzhev) [inline JSDemos/Demos/Charts/ScaleBreaks/description.md]: Again, nothing major. I only combined the last two sentences into one. 

```suggestion
You can also create custom breaks, including breaks on the [argumentAxis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/). Use the [breaks](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/breaks/) array to declare a scale break collection. Each object in this array must specify the [startValue](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/breaks/#startValue) and [endValue](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/breaks/#endValue) fields that limit a single scale break. Note that breaks smaller than two ticks might not appear on the chart.
```

File: JSDemos/Demos/Charts/ScaleBreaks/description.md [hunk 5]
```diff
@@ -1 +1,7 @@
-Scale breaks allow breaking off parts of the axis to improve the readability of a chart with high amplitude values. This example demonstrates auto-calculated scale breaks. You can change the number of breaks, their style or disabled them.
\ No newline at end of file
+Scale breaks (wavy stripes you see on the chart) cut out ranges from the value axis.  This technique may enhance chart readability if values differ greatly. As you can see in this demo, you can easily compare smaller values if scale breaks are enabled. If you disable scale breaks, the smaller values are indistinguishable.
```
Comment (vladaskorohodova) [inline JSDemos/Demos/Charts/ScaleBreaks/description.md]:
```suggestion
Scale breaks (wavy stripes you see on the chart) cut out ranges from the value axis. This technique may enhance chart readability if values differ greatly. As you can see in this demo, you can easily compare smaller values if scale breaks are enabled. If you disable scale breaks, the smaller values are indistinguishable. 
```

File: JSDemos/Demos/Charts/ScaleBreaks/description.md [hunk 6]
```diff
@@ -1 +1,7 @@
-Scale breaks allow breaking off parts of the axis to improve the readability of a chart with high amplitude values. This example demonstrates auto-calculated scale breaks. You can change the number of breaks, their style or disabled them.
\ No newline at end of file
+Scale breaks (wavy stripes you see on the chart) cut out ranges from the value axis. This technique may enhance chart readability if values differ greatly. As you can see in this demo, you can easily compare smaller values if scale breaks are enabled. If you disable scale breaks, the smaller values are indistinguishable. 
+
+The Chart component can generate scale breaks on the [valueAxis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/). The Chart detects large gaps between side-by-side points, cuts them out, and puts scale breaks instead. Scale breaks are available only for *'continuous'* or *'logarithmic'* [axis types](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/#type).
```
Comment (albertov05) [inline JSDemos/Demos/Charts/ScaleBreaks/description.md]:
```suggestion
The Chart component can generate scale breaks on the [valueAxis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/). The Chart detects large gaps between side-by-side points, cuts them out, and displays scale breaks instead. Scale breaks are available only for *'continuous'* or *'logarithmic'* [axis types](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/#type).
```

**Pull request merged**